### DISCUSSION TITLE
How to delete a route

### DISCUSSION THREAD

**Discussion by Sarabalda** [devemux86/cruiser#285] [Q&A] [answered]
Sorry for this basic question, but I have re-installed Cruiser on my phone, and now I can't find a way to cancel Start and End pins on the map, in the old version I just had to touch the point for a couple of seconds and the option "Cancel" appeared. Thank you, with your app I avoided getting lost in the wilderness a few times :)

**Comment by devemux86** [comment]
@Sarabalda thanks for the kind words!

What to do mean cancel? Close the dialog screen?

You can press the Android back button or do the back gesture.
Or can press anywhere outside the dialog window and it closes.

**Comment by devemux86** [comment, marked as answer]
If you want to delete the route, use the menu "Routes | Clear".

Or do you mean to delete a waypoint?
If the route has more than 2 waypoints, long press on a waypoint and select "Delete".

**Reply by Sarabalda** [reply]
Yes, I just wanted to delete the route. Many thanks for the prompt reply, this app is top!
